jenkins/kernel-arch-complete.sh

```diff
@@ -209,3 +209,23 @@ if [[ BUILDS_FINISHED -eq 4 ]]; then
         fi
     fi
 fi
+
+# We're only building the media tree in arm, arm64 and x86
```

Comment:
While this was true on the media-pilot branch, it's not the case any more in production.  So this "media" tree case should be moved to the list above like all the other trees.  Also I think we can send both plain text and html version of the reports.